sdk/docs/iot/README.md

```diff
@@ -1,226 +1,773 @@
-# Azure IoT Clients
+# Azure IoT Samples
```

Comment:
I don't think I agree with this change. This is the first README in `/sdk/docs/iot/` and its title is `Azure IoT Samples`. I think this should still have an intro which explains the sdk, building, etc. It's a catchall that then can feed people into specific things like tests, API docs, samples, etc.

Reply:
last commit solved this.